modules/videopress/class.videopress-gutenberg.php

```diff
@@ -46,17 +86,21 @@ public static function register_video_block_with_videopress() {
 	 *
 	 * @return string
 	 */
-	public static function render_video_block_with_videopress( $attributes, $content ) {
+	public function render_video_block_with_videopress( $attributes, $content ) {
```

Comment:
Why the change to a non-static method? There still aren't any references to `$this` in this methode, are there?

Reply:
That's true! It's unrelated to what the PR tries to solve, but decided to avoid `static` methods based on the feedback received on p7H4VZ-2bw-p2 (#comment-7379):

> Also avoiding `static` methods as much as possible if they do anything more than calculate and return a value in themselves as they can’t be mocked too.

This is actually don't doing anything more than that, but I think it's worth the change in order to have consistency with the other methods and remove any potential issue we may have if we want to add unit tests.

Reply:
Hmm, I see. I think I have yet to wrap my head around the implications for mocking (I'll probably ask on that p2 thread), but generally, if a method doesn't make use of `$this`, I think it makes sense to reflect that by making it `static`. This is my way of understanding minimalism: We don't claim to need what we don't use.

This discussion might expose wider architectural considerations -- e.g. singleton patterns (which inevitably imply `static` `init()` methods) are quite widespread in the Jetpack codebase and are somewhat hard to isolate, e.g. for unit tests -- compared e.g. to 'proper' use of OOP patterns (where constructors set up invariants that are cleaned up after by destructors) -- see our recent discussion (https://github.com/Automattic/jetpack/pull/11212#issuecomment-461770189).

I'm not feeling very strongly about this, so if you'd like to go with non-static, feel free to keep the changes.